### ISSUE TITLE
Rotation

### ISSUE THREAD

**Issue by shimiuh** [coomar2841/android-multipicker-library#117]
Can i set the rotation the image will get saved in?
I am calling to take a pic form the camera capturing a pic in portrait but you are saving the image file in landscape

**Comment by deinlandel** [comment]
Yup, experienced the same with this library. Looks like EXIF orientation is not respected.
